internal/server/polling/server.go

```diff
@@ -97,9 +99,75 @@ func (s *Server) poll(ctx context.Context, resource types.NamespacedName, secret
 		return fmt.Errorf("failed to list pull requests: %w", err)
 	}
 
+	// Log the PRs.
 	for _, pr := range prs {
 		s.log.Info("pull request", "pr", pr)
 	}
 
+	labels := map[string]string{
```

Comment:
Yes, thanks, I have not checked the PR yet, it's still in draft.

Reply:
@squaremo so, about the labels. Do you prefer to change it to smth. like that:

```
tfList, err := s.listTerraformObjects(ctx, tf, map[string]string{
    bbp.LabelKey: bbp.LabelValue,
})
```
or create the selector

```
client.MatchingLabelsSelector{
    Selector: k8sLabels.Set(labels).AsSelector(),
}
```

somewhere outside the `listTerraformObjects` method because it does not change and just pass it then to the `listTerraformObjects` function? And if the latter, where are we supposed to store such pre-created things, which are not constants? Like just create a "getter" method building and returning it?

Reply:
For now changed it to

```
tfList, err := s.listTerraformObjects(ctx, tf, map[string]string{
    bbp.LabelKey: bbp.LabelValue,
})
```